ood-portal-generator/sbin/update_ood_portal

```diff
@@ -32,6 +36,7 @@ while true; do
         -h|--help) usage ; exit 0 ; shift ;;
         -r|--rpm) RPM=1 ; shift ;;
         -f|--force) FORCE=true ; shift ;;
+        --detailed-exitcodes) CHANGE_EXIT=2 ; SKIP_EXIT=4 ; shift ;;
```

Comment:
instead of using 2 for CHANGE_EXIT what about using 3?

According to https://www.tldp.org/LDP/abs/html/exitcodes.html 2 is a reserved exit code for bash to refer to "Misuse of shell builtins".

Reply:
Made the change. I stole the numbers from Puppet as the flag `--detailed-exitcodes` will return 2 for changes. Using 3 is fine too, just need something non-zero and not one.